src/components/MenuItems/MenuItems.jsx

```diff
@@ -1,33 +1,26 @@
 import React from 'react';
-import { Redirect } from 'react-router-dom';
```

Comment:
I deleted the redirecting logic because it didn't work as it should be. @thomasdahlberg